src/analytics/analytics-tracker-window.ts

```diff
@@ -1,9 +1,56 @@
+import { isFunction } from 'lodash';
+
 export interface AnalyticsTracker {
     track(step: string, data: any): void;
-    hit(hitName: string, data: any): void;
-    hasPayloadLimit(data: any): boolean;
 }
 
 export default interface AnalyticsTrackerWindow extends Window {
     analytics: AnalyticsTracker;
+    ga(command: string, eventName: string, payload: any): void;
 }
+
+export const analyticsHit = (type: string, payload: any) => {
+    const _window = window as unknown as AnalyticsTrackerWindow;
+
+    if (!isFunction(_window?.ga)) {
+        return ;
+    }
+
+    _window?.ga('send', type, {
+        ...payload,
+        nonInteraction: false,
+    });
```

Comment:
🍹 

```suggestion
    if (isFunction(_window?.ga)) {
        _window.ga('send', type, {
            ...payload,
            nonInteraction: false,
        });
    }
```

Reply:
🍹  Or more crazy option :)

```suggestion
    _window?.ga?.('send', type, {
	       ...payload,
	       nonInteraction: false,
	  });
```

Reply:
very complicated, let's leave with isFunction :)

Reply:
done